apps/cowswap-frontend/src/modules/twap/state/twapOrdersSettingsAtom.ts

```diff
@@ -43,7 +44,7 @@ export const defaultTwapOrdersSettings: TwapOrdersSettingsState = {
 export const twapOrdersSettingsAtom = atomWithStorage<TwapOrdersSettingsState>(
   'twap-orders-settings-atom:v1',
   defaultTwapOrdersSettings,
-  createJSONStorage(() => localStorage)
+  getJotaiIsolatedStorage()
```

Comment:
Can you add that to limit orders settings as well?

Reply:
Sure